keyrings/alt/file.py

```diff
@@ -1,34 +1,36 @@
 from __future__ import with_statement
 
 import os
-import getpass
-import base64
 import sys
 import json
+import getpass
 
 from keyring.py27compat import configparser
 
 from keyring.util import properties
 from keyring.util.escape import escape as escape_for_ini
 
-from . import file_base
-
+from keyrings.alt.file_base import (
```

Comment:
You're right, of course. Will fix (..was a fallout of forking the module into another package and cherry pick the changes)